bootstrap/config/kfctl_basic_auth.yaml

```diff
@@ -40,6 +40,14 @@ components:
   - tensorboard
   - tf-job-operator
 componentParams:
+  argo:
+  - initRequired: true
```

Comment:
what's initRequired for? Why is it true?

Reply:
```
if initRequired:
  ks generate ...
else:
  ks generate
  ks param set
```

Reply:
I think this is optional param, so should be false?

Reply:
because it's optional param in prototypes, this needs to be true.  the only thing that can be false is like this:
https://github.com/kubeflow/kubeflow/blob/8e6c6cd38b55b179e169a67e59b0d6028b501875/kubeflow/pipeline/pipeline.libsonnet#L18

Reply:
IIRC optional params are not required to set at generate.
can set it by "ks param set" later

Reply:
there was a conversation such that these two are not equivalent, that is why we designed it this way.